### ISSUE TITLE
[ADD] Veterans Service Organizations to the VA service taxonomy

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#19911]
### Which Taxonomy is this term for?

VA services taxonomy

### Proposed term name/label

Veterans Service Organizations

### Proposed machine name/ API ID

veteranServiceOrganizations

### Does this overlap with any existing term in the taxonomy?

No

### Rationale for adding this term

This service was initially proposed during the initial addition of VBA service to the taxonomy. This service was recently revived by a new VBA web editor who needs to represent locations and hours for VSOs available in his Regional office building. 

It is estimated that 80% of disability and pension claims are submitted with the assistance of an accredited representative. 

Connecting Veterans to VA approved and recognized organizations and representatives has become a more visible priority because of the need to protect Veterans, service members, and their families from fraud. 

[Draft VSO service in Drupal](https://prod.cms.va.gov/health-care/veterans-service-organizations-vso)

### Which products will this term impact?

VBA Regional Office

### Have stakeholders from impacted products been consulted?

Yes

### Have you drafted content for all the relevant subfields?

Yes

### Acceptance Criteria

**Content Review and Entry**
- [x] Term is reviewed by relevant taxonomy governance body
- [x] Term is either approved for addition, or denied with an explanation
- [x] If approved, a Drupal admin from Product team may add the term to the taxonomy and save as Draft
- [x] Content for subfields is updated in Drupal by Product team
- [ ] (List any other necessary review and approval steps here)

**Technical Coordination**
- [ ] Work with Lighthouse to update the Facilities API to accept new value
  - This step can begin as soon as the API ID has been finalized and saved as Draft in Drupal 

**Launch & Change Management**
- [ ] Once Content Review and Technical Coordination are complete, term can be published in Drupal
- [ ] Update to KB articles and VBA data gathering worksheets
- [ ] Announcement to VBA editors

**Comment by mmiddaugh** [comment]
@davidmpickett would love your eyes on this when you have a moment. I'm not sure what other friendly terms might be useful. It may need some polish on the facility service header and/or description too. 
We may need CAIA eyes to quickly validate this one. 

(it uses the template without your most recent updates, fyi)

**Comment by mmiddaugh** [comment]
may need to edit [[SPLIT?] Veteran connections vs. State and local benefits & Veterans service organizations #15700](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15700#top)

**Comment by davidmpickett** [comment]
@mmiddaugh some Content feedback

**VBA description**:  I would recommend removing the line "We can also help you find state and local facilities and programs." It is not intrinsically connected to VSOs or accredited representatives.  As a Veteran who is looking to learn more about local or state benefits, I would not necessarily think to look under VSOs on a Regional Office page. I still think State and Local benefits should be a separate term.

**Facility service description**: I don't think this is necessary. The VBA description already covers these points. This is a field that is okay to leave blank. (There's a ticket in the backlog somewhere to provide better guidance on this field directly in Drupal)

**Comment by mmiddaugh** [comment]
@davidmpickett the proposed VBA description came from the `National service description - copy edited version` column of the [VBA Services taxonomy document in Sharepoint](https://dvagov.sharepoint.com/:x:/s/VBAVA.govmodernization/ET964xEqUFREirhtu34HQVIBbsNOu12QIUc-EVNF_aW98g?e=LIpNM5)

The facility service description can be removed if it isn't required.

**Comment by davidmpickett** [comment]
> [@davidmpickett](https://github.com/davidmpickett) the proposed VBA description came from the `National service description - copy edited version` column of the [VBA Services taxonomy document in Sharepoint](https://dvagov.sharepoint.com/:x:/s/VBAVA.govmodernization/ET964xEqUFREirhtu34HQVIBbsNOu12QIUc-EVNF_aW98g?e=LIpNM5)

Yes. But I'm not clear on the original provenance of that final phrase about "state and local facilities and programs".  I don't see anything like that on the [Get help from a VA accredited representative or VSO page](https://www.va.gov/get-help-from-accredited-representative/) or the [related FAQ](https://www.va.gov/resources/va-accredited-representative-faqs/). It also doesn't align with what I heard in Public Contact research.

The only place I see a connection between VSOs and state and local resources is in [this comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11267#issuecomment-1765340913) about how Vet Centers editors use the "Veteran connections" term as a place to put both.

> Vet Center editors enter details about Veteran services organizations and local/state specific resources into their "Veteran connections" local descriptions.

Which is what led to the ticket about [potentially retiring Veteran connections](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15700#top) in favor of two more specific terms. 

I think that the Vet Center conflation of these two services may have crept it's way into the VSO description for VBA and nobody caught it 18 months ago when the descriptions were copyedited (July 2023). You and I didn't have our conversation about it until at least 3 months after that (Oct 2023)

**Comment by davidmpickett** [comment]
**End of Sprint update:**
- I provided content feedback on the draft term
- Waiting for @mmiddaugh to review and respond to latest comment to close out content review portion
- @jilladams handling technical coordination with Lighthouse

**Comment by jilladams** [comment]
ID was provided to LH and we asked about their timing to add to API [here](https://dsva.slack.com/archives/C02BTJTDFTN/p1733783753629569?thread_ts=1733259911.791349&cid=C02BTJTDFTN) in Slack.

**Comment by jilladams** [comment]
VSO will be created on LH side as a benefits type service
* This will work as long as the Service is available in Drupal on a VHA facility. We do not have any of these ready to push til Jan. 
* If it’s added to a VBA facility, it won’t work, because we don’t have code to push VBA services yet.

To test VSO: Facilities can use Tugboat to set up the system service > facility service, and push it, to test with Lighthouse.

**Comment by jilladams** [comment]
re-queued this ticket for SPrint 20, to look at in Jan while Adam will have time to verify receipt.

**Comment by jilladams** [comment]
VBA definitely wants to use the term, Vet Centers may adopt. VHA did not have interest, per Justin.

**Comment by jilladams** [comment]
https://dsva.slack.com/archives/C02BTJTDFTN/p1736983372441079?thread_ts=1736270215.989329&cid=C02BTJTDFTN

1. Sitewide can publish the term now, as long as it's only selected for VBA. That keeps everything within Drupal.
2. We cannot add VHA to the term until it's in Lighthouse. It can be can be a benefit service or health service in LH (makes no difference to Sitewide).
3. Once in LH, we can add it to VHA, if Vet Centers want it. Not before.

**Comment by jilladams** [comment]
Publisehd for VBA only.
